WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift

```diff
@@ -595,6 +595,15 @@ extension MediaLibraryViewController: WPMediaPickerViewControllerDelegate {
 
         updateViewState(for: pickerDataSource.numberOfAssets())
     }
+
+    func mediaPickerController(_ picker: WPMediaPickerViewController, handleError error: Error) -> Bool {
+        let nserror = error as NSError
+        if let mediaLibrary = self.blog.media, !mediaLibrary.isEmpty {
+            let title = NSLocalizedString("Unable to Sync", comment: "Title of error prompt shown when a sync the user initiated fails.")
```

Comment:
Totally nitpicking but I think sync should be lowercase and also could we say  "Unable to  sync media"?

Reply:
I copied the string from other places is being used, in order to keep the translations.

Reply:
Oh I see